packages/dds/tree/src/test/simple-tree/unhydratedNode.spec.ts

```diff
@@ -87,19 +87,42 @@ describe("Unhydrated nodes", () => {
 	});
 
 	it("get their parent", () => {
-		const mapLeaf = new TestLeaf({ value: "mapValue" });
+		const mapLeaf = new TestLeaf({ value: "value" });
```

Comment:
Nit: there is a lot going on in this test. Would it make sense to break this up into smaller, isolated tests? If not, probably worth adding some step-by-step comments explaining what the test is doing.

Reply:
I'll add some comments!